apps/go-mod-validator/test/e2e.test.ts

```diff
@@ -22,186 +34,147 @@ vi.mock("@actions/core", async (importOriginal: any) => ({
   },
 }));
 
-function setup(repoName: string) {
-  process.env["INPUT_GO-MOD-DIR"] = join("./test/data", repoName);
+const nock = getNock("lockdown");
```

Comment:
Why passing "lockdown"?

Reply:
Oh just just to make it explicit i guess, i didnt realize its the default

Reply:
Passing it explicitly doesn't let you update the fixtures through the ENV.